core/modules/file/file.install

```diff
@@ -695,9 +740,108 @@ function file_update_1001() {
 }
 
 /**
- * Switch the 'edit files' permission from file_entity to 'manage files'.
+ * Add {file_managed}.type field
  */
 function file_update_1002() {
+  $schema = array();
+  $schema['file_metadata'] = array(
```

Comment:
Coming from Backdrop 1.13, this update hook isn't run. Why are you replacing `file_update_1002()` instead of just making this `file_update_1003()`?

Reply:
file update 1002 has been there since the very begining. This is the one that is broken for D7 sites that have file_entity module. The change here fixes it.

Reply:
ah, this is because the PR predates that update hook. fixed now.